packages/nextra-theme-docs/src/constants.tsx

```diff
@@ -142,6 +142,7 @@ export const DEFAULT_THEME: DocsThemeConfig = {
     labels: 'bug'
   },
   sidebar: {
+    defaultMenuCollapseLevel: 1,
```

Comment:
currenly setting `defaultMenuCollapseLevel` to 1 or 2 is the same, it should be fixed
```
page1
page2
page3
 - page 3-1
 - page 3-2 <- because I expected to collapse them with setting 2
   - page 3-2-1
   - page 3-2-2
```